monitoring_service.rst

```diff
@@ -2,145 +2,230 @@ Raiden Network Monitoring Service
-* locksroot (bytes32) - lock root state of the channel
-* extra_hash (bytes32) - implementation dependent extra data
-* signature (bytes32) - ecrecoverable signature of the data above, in order they are listed here
++--------------------------+------------+--------------------------------------------------------------------------------+
+| Field Name               | Field Type |  Description                                                                   |
++==========================+============+================================================================================+
+|  onchain_balance_proof   | object     | Onchain Balance Proof                                                          |
++--------------------------+------------+--------------------------------------------------------------------------------+
+|  non_closing_signature   | string     | Signature of the Onchain Balance Proof by the client                           |
++--------------------------+------------+--------------------------------------------------------------------------------+
+|  reward_amount           | uint256    | Proposed fee in RDN                                                            |
++--------------------------+------------+--------------------------------------------------------------------------------+
+|  reward_proof_signature  | string     | Signature of parts of the Onchain Balance Proof - channel_identifier,          |
+|                          |            | token_network_address, chain_id, nonce - and the reward_amount                 |
++--------------------------+------------+--------------------------------------------------------------------------------+
+
+__ https://raiden-network-specification.readthedocs.io/en/latest/smart_contracts.html#balance-proof-onchain
 
-All of this fields are required. Monitoring Service MUST perform basic verification of these data, namely channel existence. Monitoring service SHOULD accept the message if and only the sender of the message is same as the sender address recovered from the signature.
+All of this fields are required. Monitoring Service MUST perform basic verification of these data, namely channel
+existence. Monitoring service SHOULD accept the message if and only if the sender of the message is same as the sender
+address recovered from the signature.
 
 
-Example data: Balance proof
----------------------------
+Example Data: Monitor Request
+-----------------------------
 ::
 
     {
-      "nonce": 13,
-      "transferred_amount": 15000,
-      "channel_address": "0x87F5636c67f2Fd4F11710974766a5B1b6f33FB1d",
-      "extra_hash": "0xe0fa3e376941dafc9b3836f80bee307ab2eacb569ec7ccceff5e66b48b1efd9c",
-      "locksroot": "0xebd7dc7d6dd7956e62104182194939a1223c738ffc2a14dbbecb6191cf76f211",
-      "signature": "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+      "onchain_balance_proof": {
+                                 "signature_prefix": tbd,
```

Comment:
No need for so much indention. Just indent it 4 spaces.

Reply:
thank you for the detailed feedback, everything is done

Reply:
the closing bracecan be unindented as well